### DISCUSSION TITLE
Using app-specific password with get_provisioning_profile

### DISCUSSION THREAD

**Discussion by mceIdo** [fastlane/fastlane#18048] [Q&A]
I saw the documentation states you can use app specific password to upload to TestFlight, but nothing else
1. I obviously tried using the `FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD` flag anyway, but with no success - Just making sure I'm not missing an workaround to get it to work anyway
2. I cannot use App Store Connect API Key as i need it to work with an Enterprise as well
3. Wondering if there are any plans to extends the `FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD` to more commands?

**Comment by dinsen** [comment]
Short answer: It's not possible.

Only iTMSTransporter can be used with an app-specific password.

For now you need to use an Apple ID + password for it. (Not sure you can create an Auth key for Enterprise accounts yet?)

**Reply by mceIdo** [reply]
No currently Auth key is only for App Store accounts. 
So my only option at the moment is to renew the spaceship cookie once every 2-4 week, am I right?

**Reply by dinsen** [reply]
Yes 😞

**Reply by mceIdo** [reply]
All good, you saved me quite some time and frustration :)
